io.github.KikoPlayProject.KikoPlay.yml

```diff
@@ -0,0 +1,342 @@
+app-id: io.github.KikoPlayProject.KikoPlay
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: KikoPlay
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  # aria2c download
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-download
+  # sound
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --socket=x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --persist=.config/kikoplay
```

Comment:
The program should follow the XDG basedir standard, don't hardcode $HOME/.config